src/Demo.tsx

```diff
@@ -14,7 +14,10 @@ export interface DemoState {
   infoText: string;
 }
 
-export default class Demo extends React.Component<void, DemoState> {
+export default class Demo extends React.Component<any, DemoState> {
```

Comment:
Again, not sure what this is for. But it sounds like something that should maybe be in a separate place and so it's not built into the published package?

Reply:
Right. This is only included in the dev webpack build but in the production build it is not included (the entry field in the respective dev and prod files are different).